server/commands/subscriptionDestroyer.ts

```diff
@@ -0,0 +1,44 @@
+import { Transaction } from "sequelize";
+import { Event, Subscription, User } from "@server/models";
+
+type Props = {
+  /** The user destroying the subscription */
+  user: User;
+  /** The subscription to destroy */
+  subscription: Subscription;
+  /** The IP address of the user creating the subscription */
+  ip: string;
+  transaction: Transaction;
+};
+
+/**
+ * This command destroys a document subscription.
+ *
+ * This just removes the subscription.
+ * Does not touch the target document.
+ *
+ * @param Props The subscription to destroy
+ * @returns void
+ */
+export default async function subscriptionDestroyer({
```

Comment:
I'm not sure if there is a usecase for destroying the subscription completely, if you did so then the next edit of the document would just result in it becoming recreated. We likely only need to mark the subscription as disabled.